website/common/script/ops/releaseBoth.js

```diff
@@ -44,21 +46,30 @@ module.exports = function releaseBoth (user, req = {}, analytics) {
   for (animal in content.pets) {
     if (user.items.pets[animal] === -1) {
       giveTriadBingo = false;
+    } else if (user.items.pets[animal] === 0 || user.items.pets[animal] === null) {
```

Comment:
which tests failed when checking with `if(!user.items.pets[animal])`?

Now that I think about it, by default pets not owned are not stored at all inside `user.items.pets`, so the only way to check for them would be with `undefined`...